samples/vbusin/sample.yaml

```diff
@@ -11,11 +17,11 @@ common:
   timeout: 30
 
 tests:
-  sample.vbusin.eng_c:
-    extra_args: CONFIG_NPMX_DEVICE_NPM1300_ENG_C=y
+  sample.vbusin:
+    extra_args: CONFIG_NPMX_DEVICE_NPM1300=y
     harness: console
     harness_config:
-      fixture: nPM1300_setup_eng_c
+      fixture: nPM1300_setup
```

Comment:
@nordic-piks - is it ok?

Reply:
Yes, however it required adaptation on the side - I will do it now.